### PR TITLE
*: make test leak great again

### PR DESCRIPTION
**Pull request by tiancaiamao**
Fix the broken `make leak` command.

### CODE REVIEW

**Comment by zz-jason** [comment]
can we add `leak` to `dev` or `test` ?

**Comment by zimulala** [comment]
Please fix CI. @tiancaiamao

**Comment by coocood** [comment]
@tiancaiamao 
Please fix CI

**Comment by tiancaiamao** [comment]
PTAL @zz-jason @coocood

**Comment by zz-jason** [comment]
LGTM

File: store/localstore/goleveldb/goleveldb_test.go [hunk 1]
```diff
@@ -29,24 +29,26 @@ func TestT(t *testing.T) {
...
+	db        engine.DB
+	checkLeak func()
 }
 
 func (s *testSuite) SetUpTest(c *C) {
 	var (
 		d   MemoryDriver
 		err error
 	)
+	s.checkLeak = testleak.AfterTest(c)
```
Comment (zimulala) [inline store/localstore/goleveldb/goleveldb_test.go]: Can we use `testleak.BeforeTest` directly?

**Pull request merged**